app/page.tsx

```diff
@@ -76,7 +78,7 @@ async function getMsgExpRecipeCollection(): Promise<NimbusRecipeCollection> {
   return msgExpRecipeCollection
 }
 
-export default async function Dashboard() {
+export default withPageAuthRequired(async function Dashboard() {
```

Comment:
I'm inclined to instead use the first example at https://github.com/auth0/nextjs-auth0/blob/main/EXAMPLES.md#protecting-pages-with-middleware to protect all the pages, so we can't inadvertently add an unprotected route by forgetting to wrap it.  What do you think?

Reply:
I actually did try this and was having server/client issues; I'll give it another shot :)

Reply:
I found the issue, I needed to do some extra `init` things! I believe it's working with the middleware now!